test/spec/swagger/schema.js

```diff
@@ -525,3 +525,130 @@ test('support "const" keyword', async t => {
     }
   })
 })
+
+test('support "exposeHeadRoute" option', async (t) => {
```

Comment:
This test passes without the code changes on `common.js` file

Reply:
I rewrote the tests, can you please have another look?